### PR TITLE
PR: DS Linked List in C programming

### PR DESCRIPTION
**Pull request by ayushawasthi24**
# Pull Request: Linked List in C Programming Documentation

## Description

This pull request contains a new Markdown document that explains the concept of Linked Lists in C Programming. It includes a definition, an explanation in simple terms, and code snippets to help readers understand the topic.

## Changes Made

- Created a new Markdown file: `linked-list.MD`
- Added content about Linked Lists in C Programming.
- Included a clear and simple explanation for beginners.
- Provided code snippets to illustrate the concept.
- Ensured proper formatting and readability.

## Purpose

The purpose of this pull request is to contribute a beginner-friendly document about linked lists in C programming. It serves as an educational resource for those learning about data structures in C.

- Content accuracy and correctness.
- Clarity and simplicity of the explanation.
- Code snippets for accuracy and completeness.
- Proper formatting and structure of the document.

## Additional Information

This document is submitted as part of a learning initiative to make programming concepts more accessible to beginners. It aims to assist those looking to understand and work with linked lists in C.

Your feedback and suggestions for improvements are highly appreciated.

## Author

- Name: Ayush Awasthi
- Email: ayushawasthi2409@gmail.com

Thank you for considering this contribution. Your feedback and input are invaluable in improving the quality and accessibility of educational content in this project.

### CODE REVIEW

File: 1-data-structures/0-c-programming/linked-list.MD [hunk 1]
```diff
@@ -1 +1,69 @@
-linked-list
```
Comment (Thevapriyan07) [inline 1-data-structures/0-c-programming/linked-list.MD:1]: Remove this line

**Review by Thevapriyan07 — CHANGES_REQUESTED** [review]
make that change

**Comment by ayushawasthi24** [comment]
That line is already removed in the latest commit.

**Review by Thevapriyan07 — APPROVED** [review]
good

**Review by Abishethvarman — APPROVED** [review]
Approved to merge @Thevapriyan07

**Comment by Abishethvarman** [comment]
@ayushawasthi24 it would be great if you star this repository

**Pull request merged**